app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::UsersController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def index
+    if current_user
+      @current_user = current_user
+      @reviews = Review.where(user_id: @current_user.id)
+      @cities = City.where(user_id: @current_user.id)
+      render json: {current_user: @current_user, reviews: @reviews, cities: @cities}
+    else
+      render json: {current_user: nil, message: "Please sign in."}
```

Comment:
I might consider rendering a 401 unauthorized status code if we are required a user to be signed in to leverage this api endpoint.